templates/taxi/driver_list.html

```diff
@@ -5,10 +5,16 @@ <h1>Driver list</h1>
   {% if driver_list %}
     <ul>
       {% for driver in driver_list%}
-        <li>
+        {% if driver.username == user.username %}
```

Comment:
The condition to check if the driver is the current user is correctly implemented. This helps in highlighting the current user's entry in the list.